src/OnboardingSPA/steps/TheFork/index.js

```diff
@@ -133,23 +150,42 @@ const TheFork = () => {
 			/>
 			<br />
 			<br />
-			{ migrationUrl && (
-				<a
+			{ canMigrateSite ? (
+				<div
 					className="nfd-onboarding-step--site-gen__fork__importsite"
-					href={ migrationUrl }
-					target={ '_blank' }
-					rel={ 'noreferrer' }
-					onClick={ () =>
-						trackOnboardingEvent(
-							new OnboardingEvent(
-								ACTION_SITEGEN_FORK_OPTION_SELECTED,
-								'MIGRATE'
-							)
-						)
-					}
+					onClick={ () => {
+						handleMigration();
+					} }
+					onKeyUp={ ( event ) => {
+						if ( event.key === 'Enter' ) {
+							handleMigration();
+						}
+					} }
+					tabIndex={ 0 }
+					role="button"
+					aria-label="Trigger site migration"
 				>
 					{ content.importtext }
-				</a>
+				</div>
+			) : (
+				migrationUrl && (
+					<a
+						className="nfd-onboarding-step--site-gen__fork__importsite"
+						href={ migrationUrl }
+						target={ '_blank' }
+						rel={ 'noreferrer' }
+						onClick={ () =>
```

Comment:
Can we move this entire decision logic to the `handleMigration` function? If `canMigrateSite` is enabled, then `navigate`, otherwise just redirect and track the common event in both cases.